lib/codemod/package.json

```diff
@@ -7,12 +7,12 @@
     "type": "git",
     "url": "git+https://github.com/storybooks/storybook.git"
   },
-  "main": "src/index.js",
+  "main": "dist/index.js",
   "dependencies": {
     "jscodeshift": "^0.3.30"
   },
   "scripts": {
-    "prepublish": "node ../../scripts/prepublish.js && mv dist/transforms/* dist"
+    "prepublish": "node ../../scripts/prepublish.js"
```

Comment:
I updated the README for the package, was there anything else that needed to change?